extensions/search-notion/src/common/initialize.ts

```diff
@@ -0,0 +1,20 @@
+import { preferences } from "@raycast/api";
+
+export async function initialize(): Promise<{
+  cookie: string;
+  spaceID: string;
+}> {
+  const cookie = preferences.cookie?.value as
+    | string
+    | undefined;
+  if (!cookie) {
+    throw new Error("no cookie");
+  }
+  const spaceID = preferences.spaceID?.value as
+    | string
+    | undefined;
+  if (!spaceID) {
+    throw new Error("no spaceID");
+  }
+  return { cookie, spaceID };
+}
```

Comment:
This isn't necessary. We guarantee that required preferences are set. I'd recommend to use the preferences directly where they are required. BTW, checkout the [`getPreferenceValues` function](https://developers.raycast.com/api-reference/preferences#getpreferencevalues) for some easier acces.